geopandas/geoseries.py

```diff
@@ -53,6 +53,37 @@ def _geoseries_constructor_with_fallback(data=None, index=None, crs=None, **kwar
         return Series(data=data, index=index, **kwargs)
 
 
+def _geoseries_expanddim(data=None, index=None, crs=None, **kwargs):
```

Comment:
Do we pass `crs` somewhere? Because I would think that this will never be used from pandas code

Reply:
No, can remove, I was thinking about this as being "geodataframe.__init__ like", but if it's unused then it's probably clearer to remove.